engine/movie/oak_speech/oak_speech.asm

```diff
@@ -66,7 +77,7 @@ OakSpeech:
 	call PrintText
 	call GBFadeOutToWhite
 	call ClearScreen
-	ld a, NIDORINO
+	ld a, NIDORINO ; plays Nidorina's cry due to OakSpeechText2
```

Comment:
This line affects the sprite that gets displayed, not the cry that gets played.
```suggestion
	ld a, NIDORINO
```

Reply:
I'm not suggesting otherwise.